### ISSUE TITLE
Failed to connect to GeoClue2 service

### ISSUE THREAD

**Issue by exoess** [danirabbit/nimbus#143]
I started the geoclue service but no matter what i do i still get this error. i'm not on elementary os, is that maybe the issue?

**Comment by danirabbit** [comment]
Nimbus uses the location portal, so as long as your desktop environment provides that portal, it should work

**Comment by exoess** [comment]
well, I'm not using a de, I use a wayland compositor called niri. also what do you mean location portal? do you mean a desktop portal of some kind?

**Comment by danirabbit** [comment]
Yes so you need to be running one of the available XDG desktop portals, specifically one that provides the location portal: https://flatpak.github.io/xdg-desktop-portal/
